README.md

```diff
@@ -78,9 +78,11 @@ Single Message Transforms, or SMTs, can be used to customize fields or values of
     transforms.ReplaceField.type=org.apache.kafka.connect.transforms.ReplaceField$Value 
     transforms.ReplaceField.exclude=_id
     ```
+1. If you have events where there is no value ([_tombstone_ events](https://kafka.apache.org/documentation.html#compaction)), you may wish to filter these out.
+    - In the Cloudant sink connector, these may be undesirable as they will generate an empty document.
+    - In the Cloudant source connector, tombstone events are generated by default for deleted documents (in addition to the deleted document itself).
+    - In either case, you will need to use a `dropNullRecords` transform and predicate to filter out and remove these tombstone events:
```

Comment:
I realise some of this came from before, but I think the direct nature of what was stated previously was a little more suited to having only an example of removing tombstones from the sink connector. Now it is a bit more optional and the filter/predicate names are user defined anyway so it probably makes more sense to point to the predicate class name rather than the example name.
```suggestion
    - In either case, you can use the `RecordIsTombstone` predicate with a filter to remove these tombstone events as shown in this example:
```